data-service/src/main/java/com/amazonaws/blox/dataservice/model/Deployment.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may
+ * not use this file except in compliance with the License. A copy of the
+ * License is located at
+ *
+ *     http://aws.amazon.com/apache2.0/
+ *
+ * or in the "LICENSE" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+package com.amazonaws.blox.dataservice.model;
+
+import lombok.Builder;
+import lombok.Data;
+import lombok.NonNull;
+
+import java.time.Instant;
+
+@Data
+@Builder
+public class Deployment {
```

Comment:
as we talked offline, I think it depends whether we need FE to be a separate service. Otherwise this can be reduced to FE client model + DS internal model + DS DB model.

Reply:
Sounds good.